tests/core/engine_adapter/test_integration.py

```diff
@@ -884,72 +885,76 @@ def test_scd_type_2(ctx: TestContext):
 
 
 def test_sushi(ctx: TestContext):
-    if ctx.test_type == "df":
+    if ctx.test_type != "query":
```

Comment:
@treysp I realized that we don't need to run pyspark. The reason is that the way it works is that it takes the given dataframe input and converts it to pyspark (and converts it to SQL for the query tests). Since we aren't providing explicit inputs here like the other tests then we don't convert anything to pyspark so the pyspark test was just the query test run again. This also means we can remove the init since we just aren't including pyspark.